tests/acceptance/30_generic_methods/permissions_group_acl_present.cf

```diff
@@ -0,0 +1,738 @@
+
+      "files[15]"     string => "file15/.";
+      "recursive[15]" string => "true";
+      "group[15]"      string => "bin";
+      "ace[15]"       string => "-rw";
+      "create[15]"    string => "true";
+      "initial[15]"    slist => { "user:*:rwx", "group:*:rx", "group:bin:-rwx", "user:bin:r", "all:rx" };
+      "exp_lines[15]"  slist => { "user::rwx", "group::r-x", "group:bin:---", "user:bin:r", "other::r-x" };
+      "mode[15]"      string => "enforce";
+      "status[15]"    string => "success";
+      "execute[15]"   string => "true";
+
+      "files[16]"     string => "file15/subfile1";
+      "create[16]"    string => "true";
+      "initial[16]"    slist => { "user:*:rwx", "group:*:rx", "group:bin:x", "user:bin:r", "all:r" };
+      "exp_lines[16]"  slist => { "user::rwx", "group::r-x", "group:bin:--x", "user:bin:r--", "other::r--" };
+
+      "files[17]"     string => "file17/.";
+      "recursive[17]" string => "true";
+      "group[17]"      string => "bin";
+      "ace[17]"       string => "-rw";
+      "create[17]"    string => "true";
+      "initial[17]"    slist => { "user:*:rwx", "group:*:rx", "group:bin:x", "user:bin:r", "all:rx" };
+      "exp_lines[17]"  slist => { "user::rwx", "group::r-x", "group:bin:--x", "user:bin:r--", "other::r-x" };
+      "mode[17]"      string => "enforce";
+      "status[17]"    string => "success";
+      "execute[17]"   string => "true";
+
+      # non-recursive
+      "files[18]"     string => "file18";
+      "recursive[18]" string => "false";
+      "group[18]"      string => "*";
+      "ace[18]"       string => "+rx";
+      "create[18]"    string => "true"; 
+      "initial[18]"    slist => { "user:*:rwx", "group:*:rx", "group:bin:r", "user:bin:r", "all:r" };
+      "exp_lines[18]"  slist => { "user::rwx", "group::r-x", "group:bin:r--", "user:bin:r--", "other::r--" };
+      "mode[18]"      string => "enforce";
+      "status[18]"    string => "success";
+      "execute[18]"   string => "true";
+
+      "files[19]"     string => "file19";
+      "recursive[19]" string => "false";
+      "group[19]"      string => "bin";
+      "ace[19]"       string => "=rw";
+      "create[19]"    string => "true";
+      "initial[19]"    slist => { "user:*:rwx", "group:*:rx", "group:bin:rw", "user:bin:r", "all:r" };
+      "exp_lines[19]"  slist => { "user::rwx", "group::r-x", "group:bin:rw", "user:bin:r", "other::r" };
+      "mode[19]"      string => "enforce";
+      "status[19]"    string => "success";
+      "execute[19]"   string => "true";
+
+      "files[20]"     string => "file20";
+      "recursive[20]" string => "false";
+      "group[20]"      string => "bin";
+      "ace[20]"       string => "-rw";
+      "create[20]"    string => "true";
+      "initial[20]"    slist => { "user:*:rwx", "group:*:rx", "group:bin:-rwx", "user:bin:r", "all:r" };
+      "exp_lines[20]"  slist => { "user::rwx", "group::r-x", "group:bin:---", "user:bin:r--", "other::r--" };
```

Comment:
why is bin loosing the x ?
and how can it be success ?

Reply:
same as the first comment